Description: Fix typos and a missing reference.
 Fixes typos in the "black hat" and "I for one welcome our new X overlords" entries.
 Adds a missing sense number of a "ACK" reference in the "NAK" entry.
Author: Oohara Yuuma <oohara@libra.interq.or.jp>
Bug-Debian: https://bugs.debian.org/581692
Bug-Debian: https://bugs.debian.org/587744
Bug-Debian: https://bugs.debian.org/587745
Reviewed-By: Ruben Molina <ruben.molina@udea.edu.co>
Forwarded: no
Last-Update: 2015-09-03

--- dict-jargon-4.4.7.orig/jargon.xml
+++ dict-jargon-4.4.7/jargon.xml
@@ -9651,7 +9651,7 @@ bits     0001  0011  0101  0111  1001  1
 <glossdef>
    <para> 1. [common among security specialists] A
    <glossterm>cracker</glossterm>, someone bent on breaking into the system
-   you are protecting.  Oppose the less comon <emphasis role='firstterm'>white
+   you are protecting.  Oppose the less common <emphasis role='firstterm'>white
    hat</emphasis> for an ally or friendly security specialist; the term
    <emphasis role='firstterm'>gray hat</emphasis> is in occasional use for
    people with cracker skills operating within the law, e.g. in doing security
@@ -37740,7 +37740,7 @@ would be read as &lsquo;foobaz&rsquo; or
    <citetitle>Simpsons</citetitle> episode <ulink
    url='http://www.tvtome.com/tvtome/servlet/GuidePageServlet/showid-146/epid-1381/'>(#96,
    <citetitle>Homer In Space</citetitle>)</ulink> X = <quote>insect</quote>
-   and th line is part of a speech in which a smarmy newscaster expresses his
+   and the line is part of a speech in which a smarmy newscaster expresses his
    willingness to collaborate with an invading race of giant space
    ants.</para>
 </glossdef>
@@ -48191,7 +48191,7 @@ retorted, <quote>Mu!</quote> </para></bl
 <glossdef>
    <para> 3. Used to politely interrupt someone to tell them you don't
    understand their point or that they have suddenly stopped making sense.
-   See <glossterm>ACK</glossterm>, sense </para>
+   See <glossterm>ACK</glossterm>, sense 3.</para>
 </glossdef>
 <glossdef>
    <para> 3.  <quote>And then, after we recode the project in
